Fix TestISequencer_Flush()

https://github.com/voedger/voedger/actions/runs/14610472602/job/40987449667
```
==================
  WARNING: DATA RACE
  Read at 0x00c0005763c8 by goroutine 170960:
    github.com/voedger/voedger/pkg/isequencer.(*MockStorage).ReadNextPLogOffset()
        /home/runner/work/voedger/voedger/pkg/isequencer/test_utils.go:168 +0x244
    github.com/voedger/voedger/pkg/isequencer.(*sequencer).actualizer.func2()
        /home/runner/work/voedger/voedger/pkg/isequencer/impl.go:403 +0x6a
    github.com/voedger/voedger/pkg/coreutils.Retry()
        /home/runner/work/voedger/voedger/pkg/coreutils/retry.go:20 +0xe6
    github.com/voedger/voedger/pkg/isequencer.(*sequencer).actualizer()
        /home/runner/work/voedger/voedger/pkg/isequencer/impl.go:402 +0x264
    github.com/voedger/voedger/pkg/isequencer.(*sequencer).startActualizer.gowrap1()
        /home/runner/work/voedger/voedger/pkg/isequencer/impl.go:76 +0x4f
  
  Previous write at 0x00c0005763c8 by goroutine 170956:
    github.com/voedger/voedger/pkg/isequencer.(*MockStorage).WriteValuesAndNextPLogOffset()
        /home/runner/work/voedger/voedger/pkg/isequencer/test_utils.go:148 +0x56d
    github.com/voedger/voedger/pkg/isequencer.(*sequencer).flusher.func1()
        /home/runner/work/voedger/voedger/pkg/isequencer/impl.go:154 +0x93
    github.com/voedger/voedger/pkg/coreutils.Retry()
        /home/runner/work/voedger/voedger/pkg/coreutils/retry.go:20 +0xe6
    github.com/voedger/voedger/pkg/isequencer.(*sequencer).flusher()
        /home/runner/work/voedger/voedger/pkg/isequencer/impl.go:153 +0x4fa
    github.com/voedger/voedger/pkg/isequencer.(*sequencer).startFlusher.gowrap1()
        /home/runner/work/voedger/voedger/pkg/isequencer/impl.go:68 +0x4f
  
  Goroutine 170960 (running) created at:
    github.com/voedger/voedger/pkg/isequencer.(*sequencer).startActualizer()
        /home/runner/work/voedger/voedger/pkg/isequencer/impl.go:76 +0x1ad
    github.com/voedger/voedger/pkg/isequencer.(*sequencer).Actualize()
        /home/runner/work/voedger/voedger/pkg/isequencer/impl.go:469 +0x24d
    github.com/voedger/voedger/pkg/isequencer.New()
        /home/runner/work/voedger/voedger/pkg/isequencer/provide.go:57 +0x65e
    github.com/voedger/voedger/pkg/isequencer_test.TestISequencer_Flush.func2.4()
        /home/runner/work/voedger/voedger/pkg/isequencer/isequencer_test.go:435 +0xd0
    testing.tRunner()
        /opt/hostedtoolcache/go/1.24.2/x64/src/testing/testing.go:1792 +0x225
    testing.(*T).Run.gowrap1()
        /opt/hostedtoolcache/go/1.24.2/x64/src/testing/testing.go:1851 +0x44
  
  Goroutine 170956 (finished) created at:
    github.com/voedger/voedger/pkg/isequencer.(*sequencer).startFlusher()
        /home/runner/work/voedger/voedger/pkg/isequencer/impl.go:68 +0x1a4
    github.com/voedger/voedger/pkg/isequencer.(*sequencer).actualizer()
        /home/runner/work/voedger/voedger/pkg/isequencer/impl.go:398 +0x19c
    github.com/voedger/voedger/pkg/isequencer.(*sequencer).startActualizer.gowrap1()
        /home/runner/work/voedger/voedger/pkg/isequencer/impl.go:76 +0x4f
  ==================
  --- FAIL: TestLongRace (206.67s)
      --- FAIL: TestLongRace/panic_on_wrong_usage#1[134](https://github.com/voedger/voedger/actions/runs/14610472602/job/40987449667#step:7:136) (0.00s)
          --- FAIL: TestLongRace/panic_on_wrong_usage#1134/after_cleanup (0.00s)
              testing.go:1490: race detected during execution of test
```

--- a/pkg/isequencer/test_utils.go
+++ b/pkg/isequencer/test_utils.go
@@ -98,9 +98,6 @@ func (m *MockStorage) ReadNumbers(wsid WSID, seqIDs []SeqID) ([]Number, error) {
 
 	result := make([]Number, len(seqIDs))
 
-	m.numbersMu.RLock()
-	defer m.numbersMu.RUnlock()
-
 	wsNumbers, exists := m.Numbers[wsid]
 	if !exists {
 		return result, nil // Return zeros if workspace not found
@@ -119,8 +116,8 @@ func (m *MockStorage) ReadNumbers(wsid WSID, seqIDs []SeqID) ([]Number, error) {
 func (m *MockStorage) WriteValuesAndNextPLogOffset(batch []SeqValue, offset PLogOffset) error {
 	// notest
 
-	m.mu.RLock()
-	defer m.mu.RUnlock()
+	m.mu.Lock()
+	defer m.mu.Unlock()
 
 	if m.onWriteValuesAndOffset != nil {
 		m.onWriteValuesAndOffset()
@@ -129,9 +126,6 @@ func (m *MockStorage) WriteValuesAndNextPLogOffset(batch []SeqValue, offset PLog
 	if m.writeValuesAndOffsetError != nil {
 		return m.writeValuesAndOffsetError
 	}
-
-	m.numbersMu.Lock()
-	defer m.numbersMu.Unlock()
 
 	// batch could be empty here for the offset that is just written
 	// case:

--- a/pkg/isequencer/types.go
+++ b/pkg/isequencer/types.go
@@ -109,7 +109,6 @@ type MockStorage struct {
 	ReadNumbersError          error
 	writeValuesAndOffsetError error
 	mu                        sync.RWMutex
-	numbersMu                 sync.RWMutex
 	pLog                      map[PLogOffset][]SeqValue // Simulated PLog entries
 	readNextOffsetError       error
 	onWriteValuesAndOffset    func()

